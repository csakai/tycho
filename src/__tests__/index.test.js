import React from 'react';
import toJson from 'enzyme-to-json';
import index from '../index.js';

jest.mock('../components/App');

describe('Main Entry', () => {
  it('should render without crashing', () => {
    expect(toJson(index)).toMatchSnapshot();
  });
});
