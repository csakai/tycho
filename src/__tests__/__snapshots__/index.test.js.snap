// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Entry should render without crashing 1`] = `undefined`;
